scripts/send_dos_opportunities_email.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+"""
+Description:
+    For every lot the script will:
+        - look for latest briefs for Digital Outcomes and Specialists
+        - if there are new briefs on that lot:
+            - it will create a Mailchimp campaign
+            - it will set the content of the campaign to be the briefs found
+            - it will send that campaign to the list_id as referenced in the `lots` variable
+        - if there are no new briefs on that lot no campaign is created or sent
+
+    Number of days tells the script for how many days before current date it should include in its search for briefs
+        example:  if you run this script on a Wednesday and set number of days=1,
+                    then it will only include briefs from Tuesday (preceding day).
+        example2: if you run it on Wednesday and set number of days=3,
+                    then it will include all briefs published on Sunday, Monday and Tuesday.
+
+    For testing purposes, you can override the list ID so you can send it to yourself only as
+    we have set up a testing list with ID "096e52cebb"
+
+Usage:
+    send_dos_opportunities_email.py
+        <stage> <api_token> <mailchimp_username> <mailchimp_api_key> <number_of_days> --list_id=<list_id>
+
+Example:
+    send_dos_opportunities_email.py
+        preview b7g5r7e6gv876tv6 user@gds.gov.uk 7483crh87h34c3 3 --list_id=988972hse
+"""
+
+import sys
+
+from docopt import docopt
+from mailchimp3 import MailChimp
+from dmapiclient import DataAPIClient
+
+sys.path.insert(0, '.')
+from dmscripts.helpers.env_helpers import get_api_endpoint_from_stage
+from dmscripts.send_dos_opportunities_email import main
+
+
+lots = [
+    {
+        "lot_slug": "digital-specialists",
```

Comment:
I'd probably propose a slightly different route here
1.  Not updating these dicts with the list ID: The list ID doesn't bear any reference to the lots as far as I can see. Therefore calling the list `lots` is a bit misleading.
2. Not having a placeholder: Theres no need for the placeholder as both keys and values can be dynamically updated at any point. It doesn't serve any purpose really.

Maybe something like:
```
lots = (
    ("digital-specialists", "Digital specialists"),
    ("digital-outcomes", "Digital outcomes"),
    ("user-research-participants", "User research participants")
)
...
...
for lot_slug, lot_name in lots:
    ok = main(
        data_api_client=data_api_client,
        mailchimp_client=mailchimp_client,
        lot_slug=lot_slug,
        lot_name=lot_name,
        list_id=list_id,
        number_of_days=int(arguments['<number_of_days>'])
    )
```